localstack/services/lambda_/invocation/execution_environment.py

```diff
@@ -156,7 +156,7 @@ def get_environment_variables(self) -> Dict[str, str]:
             env_vars["_HANDLER"] = self.function_version.config.handler
         # Not defined for custom runtimes (e.g., provided, provided.al2)
```

Comment:
I think that it may not be propagated to the child processes of the init, as the runtime sets it itself, and the extensions on AWS do not have it set at all. So I think the comment is not valid here anymore, missed it!